spec/lib/local-links-manager/export/bad_links_url_and_status_exporter_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+require 'local-links-manager/export/bad_links_url_and_status_exporter'
+
+describe LocalLinksManager::Export::BadLinksUrlAndStatusExporter do
+  let(:exporter) { LocalLinksManager::Export::BadLinksUrlAndStatusExporter }
+
+  before do
+    create(:local_authority, homepage_url: "http://www.hogsmeade.gov.uk", status: "403")
```

Comment:
Thanks!  I've broken the before blocks out and updated links_status_exporter_spec.